lib/AddressFieldGroup/AddressEdit/EmbeddedAddressForm.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { Field } from 'redux-form';
+import { Field, ReduxFormContext } from 'redux-form';
```

Comment:
You can zap `ReduxFormContext` here now that you moved it into the wrapper.

Reply:
Thanks for catching this!